exasol/extension_wrapper_common.py

```diff
@@ -60,8 +61,8 @@ def encapsulate_bucketfs_credentials(
         IDENTIFIED BY {{BUCKETFS_PASSWORD!s}}
     """
     query_params = {
-        "BUCKETFS_USER": conf.BUCKETFS_USER,
-        "BUCKETFS_PASSWORD": conf.BUCKETFS_PASSWORD,
+        "BUCKETFS_USER": conf.get(CKey.bfs_user),
+        "BUCKETFS_PASSWORD": conf.get(CKey.bfs_password),
```

Comment:
If you need to convert settings to query parameters like this:
```python
        "BUCKETFS_USER": conf.get(CKey.bfs_user),
        "BUCKETFS_PASSWORD": conf.get(CKey.bfs_password),
```
You could consider adding something like this:
```python
def _query_parameters(config, keys):
    return { key.value: config.get(key) for key in keys }
    
    
query = params = _query_parameters(conf, [CKey.bfs_user, CKey.bfs_password])
```

Reply:
Maybe a good idea for a bigger set of parameters. I think it is more readable as it is now. If we use the enum name as the parameter name we need to insert it into the SQL statement. Not impossible, but it won't make it look nicer.